samsara.gemspec

```diff
@@ -22,11 +22,13 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "activerecord",  ">= 3.0"
-  spec.add_dependency "activesupport", ">= 3.0"
+  spec.add_dependency "activerecord",  ">= 5.2"
```

Comment:
We are the only ones using this right, since it is open source...

Reply:
yeah, as far as i know, we are